backend/common/src/main/java/ai/verta/modeldb/common/config/NFSEndpointConfig.java

```diff
@@ -1,13 +1,21 @@
 package ai.verta.modeldb.common.config;
 
 public class NFSEndpointConfig {
-  public String getArtifact;
-  public String storeArtifact;
+  private String getArtifact;
+  private String storeArtifact;
 
   public void Validate(String base) throws InvalidConfigException {
     if (getArtifact == null || getArtifact.isEmpty())
       throw new InvalidConfigException(base + ".getArtifact", Config.MISSING_REQUIRED);
     if (storeArtifact == null || storeArtifact.isEmpty())
       throw new InvalidConfigException(base + ".storeArtifact", Config.MISSING_REQUIRED);
   }
+
+  public String getGetArtifact() {
```

Comment:
typo: should be `getArtifact`

Reply:
it is not typo, the field name is `GetArtifact` so it has two times get there :)